front/app/components/admin/PostManager/components/PostTable/Row/StyledRow.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// components
+import { Tr } from 'components/admin/Table';
+
+import { Props as RowProps } from 'components/admin/Table/Tr';
+
+const StyledRow = styled(Tr)<{ undraggable: boolean }>`
```

Comment:
I believe `styled-components` automatically forwards all other props so this type argument is only for your own sanity if you want to use any of the props inside of the style definition. Since we're not using the other types I think it's ok like this